tomviz/ThreadedExecutor.h

```diff
@@ -0,0 +1,35 @@
+/* This source file is part of the Tomviz project, https://tomviz.org/.
+   It is released under the 3-Clause BSD License, see "LICENSE". */
+
+#ifndef tomvizThreadedExecutor_h
+#define tomvizThreadedExecutor_h
+
+#include <QObject>
+
+#include "PipelineExecutor.h"
+
+namespace tomviz {
+class DataSource;
+class Operator;
+class Pipeline;
+
```

Comment:
Is this "The default pipeline executor, providing execution of pipelines in a background thread in order to retain interactivity in the user interface."

Reply:
Done